pkgs/development/libraries/rocksdb/default.nix

```diff
@@ -1,19 +1,42 @@
-{ stdenv, fetchFromGitHub, lib, bzip2, cmake, lz4, snappy, zlib, zstd, enableLite ? false }:
+{ stdenv, fetchFromGitHub, fetchpatch
+, cmake
+, bzip2, lz4, snappy, zlib, zstd
+, enableLite ? false
+}:
 
 stdenv.mkDerivation rec {
   pname = "rocksdb";
-  version = "6.2.2";
+  version = "6.2.4";
```

Comment:
I think a point release like this is fine for stable? Also, very little changed in practice.